plugins/meta-inject/impl/src/test/java/org/pentaho/di/trans/steps/metainject/MetaInjectMigrationTest.java

```diff
@@ -40,9 +40,9 @@ public void testMigration() {
     String[] oldMappingNames = { "SCHENAMENAMEFIELD", "DATABASE_FIELDNAME", "STREAM_FIELDNAME", "DATE_RANGE_START_FIELD" };
     //The expected new mappings. Since the migration process removes and adds again the new  mapping, the one that does
     // not need change will become the first one.
-    String[] newMappingNames = { "DATABASE_FIELD_NAME", "DATE_RANGE_START_FIELD", "SCHEMANAMEFIELD", "DATABASE_STREAM_NAME" };
+    String[] newMappingNames = { "DATE_RANGE_START_FIELD", "SCHEMANAMEFIELD" ,"DATABASE_FIELD_NAME", "DATABASE_STREAM_NAME" };
```

Comment:
FYI, this was a change that somehow got lost when merging in a recent master. Only found by running the tests.